### PR TITLE
Users must complete registration and signup forms

### PR DESCRIPTION
**Pull request by paymog**
User can't leave any fields blank when signing in or registering

### CODE REVIEW

**Comment by paymog** [comment]
Not sure if github is messing up but I don't see what lines you made those comments on. Here's what I see:
screen shot 2015-02-09 at 8 58 15 pm

**Comment by kjrcda** [comment]
oh try going into the actual commit i think they will be displayed inline there

**Comment by paymog** [comment]
They weren't. Lemme try with another browser.

**Comment by paymog** [comment]
Yeah, don't see them there either. Even with Safari.

File: webapp/public/javascripts/controllers.js [hunk 1]
```diff
@@ -10,6 +10,11 @@ lunchrControllers.controller('MainPageController', ['$scope', '$http', '$locatio
 
         $scope.logIn = function () {
             $scope.errorsMessages = null;
+
+            if(!$scope.email || !$scope.password) {
+                return
```
Comment (kyleleung75) [inline webapp/public/javascripts/controllers.js]: @paymog Josh was referring to the missing semi colon here or the semi colon on line 47

Comment (kjrcda) [inline webapp/public/javascripts/controllers.js]: @paymog oh ya sorry I meant to show you earlier. not sure why it didn't show. line 15 and 44

**Comment by paymog** [comment]
This shouldn't be merged until #16 is merged. After which, this needs to be updated for reflect testing.

**Pull request closed without merging**